### ISSUE TITLE
Path doesn't re-render correctly when `fill` prop changes

### ISSUE THREAD

**Issue by clarkg** [software-mansion/react-native-svg#2535]
### Description

Path doesn't re-render correctly when `fill` prop changes

Reproed on React Native new Arch on iOS

### Steps to reproduce

1.
```
const ReproComponent = () => {
const [color, setColor] = useState("#0000FF");

return  <>
<Svg width={12} height={15} viewBox="0 0 12 15" fill="none">
    <Path
      d="M1 3.353C1 2.053 2.053 1 3.353 1H8.647C9.947 1 11 2.053 11 3.353V13.207C11 13.3008 10.9736 13.3927 10.9238 13.4722C10.874 13.5518 10.8029 13.6157 10.7185 13.6567C10.6341 13.6976 10.5399 13.7141 10.4466 13.7041C10.3533 13.694 10.2648 13.658 10.191 13.6L7.202 11.25C6.99452 11.0876 6.7385 10.9995 6.475 11H3.353C2.72895 11 2.13045 10.7521 1.68918 10.3108C1.2479 9.86955 1 9.27105 1 8.647V3.353Z"
      stroke={color}
      fill={color}
      strokeOpacity={0.5}
      strokeWidth={1.5}
    />
  </Svg>
  <TouchableOpacity onPress={setColor("none")}>Click to remove color</TouchableOpacity>
  </>;
  }
  
  
  ```
  
  2. 

### Snack or a link to a repository

https://gist.github.com/clarkg/b3cf2f50403f7e436bccb24cb55fe90b

### SVG version

15.8.0

### React Native version

0.76.0

### Platforms

iOS

### JavaScript runtime

Hermes

### Workflow

Expo Dev Client

### Architecture

Fabric (New Architecture)

### Build type

Debug app & dev bundle

### Device

iOS simulator

### Acknowledgements

Yes

**Comment by jakex7** [comment]
Hey @clarkg, thanks for bringing that up. Just to clarify, the issue only occurs when setting the value to `'none'`. It will be fixed in #2536.

**Comment by clarkg** [comment]
tyvm for the prompt fix @jakex7 !
